controllers/src/test/scala/cromwell/pipeline/controller/RunControllerTest.scala

```diff
@@ -34,102 +34,144 @@ class RunControllerTest
+        when(runService.getRunByIdAndUser(runId, projectId, accessToken.userId))
+          .thenReturn(Future.successful(runRespOption))
 
-        Get(s"/runs/$runId") ~> runController.route(accessToken) ~> check {
+        Get(s"/projects/${projectId.value}/runs/$runId") ~> runController.route(accessToken) ~> check {
           status shouldBe StatusCodes.OK
           responseAs[Option[Run]] shouldEqual runRespOption
         }
       }
 
       "returns the internal server error if service fails" taggedAs Controller in {
-        val runId = TestRunUtils.getDummyRunId
-        when(runService.getRunByIdAndUser(runId, accessToken.userId))
+        val dummyRun: Run = TestRunUtils.getDummyRun()
+        val runId = dummyRun.runId
+        val projectId = dummyRun.projectId
+        when(runService.getRunByIdAndUser(runId, projectId, accessToken.userId))
           .thenReturn(Future.failed(new RuntimeException("something went wrong")))
 
-        Get(s"/runs/$runId") ~> runController.route(accessToken) ~> check {
+        Get(s"/projects/${projectId.value}/runs/$runId") ~> runController.route(accessToken) ~> check {
           status shouldBe StatusCodes.InternalServerError
         }
       }
 
       "returns NotFound if the run id is not valid" taggedAs Controller in {
+        val projectId = TestRunUtils.getDummyProjectId
         val invalidRunId = "123"
-        Get(s"/runs/$invalidRunId") ~> Route.seal(runController.route(accessToken)) ~> check {
+
+        Get(s"/projects/${projectId.value}/runs/$invalidRunId") ~> Route.seal(runController.route(accessToken)) ~> check {
           status shouldBe StatusCodes.NotFound
         }
       }
     }
 
+    "get runs by project" should {
+
+      "returns all runs of a project" taggedAs Controller in {
+        val dummyRun1 = TestRunUtils.getDummyRun()
+        val projectId = dummyRun1.projectId
+        val dummyRun2: Run = TestRunUtils.getDummyRun(projectId = projectId)
+        val runRespSeq: Seq[Run] = Seq(dummyRun1, dummyRun2)
+
+        when(runService.getRunsByProject(projectId, accessToken.userId)).thenReturn(Future.successful(runRespSeq))
+
+        Get(s"/projects/${projectId.value}/runs") ~> runController.route(accessToken) ~> check {
+          status shouldBe StatusCodes.OK
+          responseAs[Seq[Run]] shouldEqual runRespSeq
+        }
+      }
+
+      "returns the internal server error if service fails" taggedAs Controller in {
+        val dummyRun: Run = TestRunUtils.getDummyRun()
+        val projectId = dummyRun.projectId
+        when(runService.getRunsByProject(projectId, accessToken.userId))
+          .thenReturn(Future.failed(new RuntimeException("something went wrong")))
+
+        Get(s"/projects/${projectId.value}/runs") ~> runController.route(accessToken) ~> check {
+          status shouldBe StatusCodes.InternalServerError
+        }
+      }
+
```

Comment:
Can you not leave unused lines, and then the next time they are formatted, they will fall into the new MR